src/zalopay/__tests__/__mocks__/zod/responses.ts

```diff
@@ -0,0 +1,13 @@
+export const createInvoiceSuccess = {
+  orderUrl: "https://zlpqc-onlpm-zod.zalopay.vn/zod/0a553-230525_77222"
+};
+
+export const queryInvoiceSuccess = {
+  orderUrl: "https://zlpqc-onlpm-zod.zalopay.vn/zod/0a553-230525_77222"
+};
+
+export const queryStatusSuccess = {
+  amount: 430000,
+  status: 1,
```

Comment:
I see that status field is string type in the specs.
But value here is number. If string should use "" for precise

Reply:
I have checked with ZOD implementing team, the amount field is a string and the status field is a number.
I will update these on our docs